src/api/comment/content-types/comment/schema.json

```diff
@@ -24,7 +24,7 @@
     "author": {
       "type": "string"
     },
-    "user": {
```

Comment:
I think you need to rebase. https://github.com/dev-launchers/strapiv4/commit/2f65c8f99e63fbfc20c11b5d23893175786ec181 changed this to user, but you are reverting back to `authorId`.

Reply:
I added the rebase

Reply:
> I think you need to rebase. [2f65c8f](https://github.com/dev-launchers/strapiv4/commit/2f65c8f99e63fbfc20c11b5d23893175786ec181) changed this to user, but you are reverting back to `authorId`.

@chungthuang so this the reason some tests are not passing ?

Reply:
Yes, you should rebase from the main branch. https://github.com/dev-launchers/strapiv4/actions/runs/10159205533/job/28092924319#step:6:13 shows the test failed because of `Error: inversedBy attribute subscriptions not found target plugin::users-permissions.user`, so it's likely because you didn't rebase.